packages/material/src/controls/MaterialInputControl.tsx

```diff
@@ -91,12 +91,13 @@ export abstract class MaterialInputControl extends Control<ControlProps & WithIn
             isValid={isValid}
             visible={visible}
           />
+          {description !== undefined ? (
+            <FormHelperText>
+              {showDescription || !isValid ? description : null}
```

Comment:
why check for an invalid state here? I think it makes sense to only check for showDescription

Reply:
I kinda agree. Both `FormHelperText`s should behave the same way: ither include them both and let their children be `null` or let check them condtionaly